README.md

```diff
@@ -2,54 +2,79 @@
 
 # NGINX Modern Reference Architectures
 
-This repository has the basics for a common way to deploy and manage a modern app. Over time we'll be building more 
-example architectures using different deployment models and options including other clouds, and you’ll be able to find
-those from here.
+A repository for modern application deployment and management, 
+created to illustrate modern app design principles and encourage 
+our community to deploy on Kubernetes. This project provides 
+a complete, stealable, easy-to-deploy, and standalone example 
+of how a modern app architecture can be built.
+
+
 
 ## Modern App Architectures
-We define modern app architectures as those driven by four characteristics: *scalability*, *portability*, *resiliency*,
-and *agility*. While of course there are many different aspects of a modern architecture, these are fundamental.
+While many different aspects of modern architectures exist, 
+we define them as driven by four fundamental characteristics: 
+*scalability*, *portability*, *resiliency*, and *agility*. 
+
+* **Scalability** – Quickly and seamlessly scale up or down to accommodate spikes or reductions in demand, anywhere in the world.
+
+* **Portability** – Easy to deploy on multiple types of devices and infrastructures, on public clouds, and on premises.
+
+*	**Resiliency** – Can fail over to newly spun‑up clusters or virtual environments in different availability regions, clouds, or data centers.
+
+*	**Agility** – Ability to update through automated CI/CD pipelines with higher code velocity and more frequent code pushes.
```

Comment:
Both **Agility** and **Resiliency** have extra spaces between the bullet and the word, I would just even those out.

Reply:
Oh interesting, didn't notice since it looks normal in preview – will fix that. Was also curious about the single line breaks in the terminal markup, totally makes sense for scrolling.

Reply:
Many developers are reading the document in a pure text mode, so it matters how the text is rendered in github **and** how it is displayed as plain text.

Reply:
Got it, I'll keep that in mind.